frontend/src/cljs/imas_seamap/db.cljs

```diff
@@ -3,10 +3,10 @@
 ;;; Released under the Affero General Public Licence (AGPL) v3.  See LICENSE file for details.
 (ns imas-seamap.db)
 
-(goog-define api-url-base "http://localhost:8000/api/")
-(goog-define media-url-base "http://localhost:8000/media/")
-(goog-define wordpress-url-base "http://localhost:8888/")
-(goog-define img-url-base "/img/")
+(def api-url-base       "http://localhost:8000/api/")
+(def media-url-base     "https://data-dev.imas.utas.edu.au/seamap/media/")
+(def wordpress-url-base "https://seamapaustralia-dev.imas.utas.edu.au/")
+(def img-url-base       "/app/img/")
```

Comment:
Any reason these don't default to localhost now?  (just from a consistency/documentation standpoint)

Reply:
Shoot, because I made a mistake, whoops. Fixed now.